### PR TITLE
Creating zip, rpm, deb, exe, tar packages

### PR DESCRIPTION
**Pull request by ShamsUlAzeem**

### CODE REVIEW

File: konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/ServeCommand.java [hunk 1]
```diff
@@ -321,28 +321,45 @@ private void addJavaCommand(List<String> cmd) {
...
-            IOUtils.copy(is, os);
-        } catch (IOException e){
-            log.error("Error extracting logback file: file does not exist or temp directory cannot be written to?", e);
+        File out = new File(FileUtils.getTempDirectory(), name);
+        File inputFile = new File(file);
+        if(inputFile.exists() && inputFile.isFile()) {
+            return inputFile;
+        }
+        else {
+            try(InputStream is = new ClassPathResource(file).getInputStream();
```
Comment (agibsonccc) [inline konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/ServeCommand.java:354]: You can create multiple try with resources with:
try(YourResource resource = new Resource(); SecondResource resource = new Resource()) {
...
}

Mind implementing this?

File: konduit-serving-deb/pom.xml [hunk 2]
```diff
@@ -14,10 +14,10 @@
     <name>konduit-serving-deb</name>
     <properties>
         <jdeb.signPackage>false</jdeb.signPackage>
-        <prefix>/opt/konduit/serving</prefix>
+        <prefix>/opt/konduit</prefix>
```
Comment (agibsonccc) [inline konduit-serving-deb/pom.xml:17]: Why remove the sub folder?

File: konduit-serving-tar/src/main/assembly/assembly.xml [hunk 3]
```diff
@@ -30,7 +30,7 @@
 
     <files>
         <file>
-            <source>${konduit-serving.root.dir}/konduit-serving-tar/target/konduit-serving-dist-${project.version}.jar</source>
+            <source>${konduit-serving.root.dir}/konduit-serving-uberjar/target/konduit-serving-uberjar-${project.version}-${spin.version}-${javacpp.platform}-${device}.jar</source>
```
Comment (agibsonccc) [inline konduit-serving-tar/src/main/assembly/assembly.xml:33]: Did you update the uber jar deps?

**Review by agibsonccc — CHANGES_REQUESTED** [review]
Minor changes

File: konduit-serving-deb/pom.xml (hunk 2 above)
Comment (ShamsUlAzeem) [reply konduit-serving-deb/pom.xml:17]: For consistency across platforms. The tar and zip packages also contains everything in the konduit root folder

File: konduit-serving-tar/src/main/assembly/assembly.xml (hunk 3 above)
Comment (ShamsUlAzeem) [reply konduit-serving-tar/src/main/assembly/assembly.xml:33]: Yup

**Review by agibsonccc — APPROVED** [review]

**Pull request merged**